src/services/tx/txEvents.ts

```diff
@@ -27,6 +28,7 @@ type Id = { txId: string; groupKey?: string } | { txId?: string; groupKey: strin
 type HumanDescription = { humanDescription?: string }
 
 interface TxEvents {
+  [TxEvent.USER_QUIT]: undefined
```

Comment:
Maybe it should be undefined, all of the other events have details. Maybe an empty object?

Reply:
Addressed in aec7801d2a6e83ca98b0a4588f6e52927ed59792.